contracts/campaign/Campaign.sol

```diff
@@ -0,0 +1,70 @@
+// SPDX-License-Identifier: GPL-3.0
+pragma solidity 0.8.7;
+
+import {SafeMath} from "@openzeppelin/contracts/utils/math/SafeMath.sol";
+import {IERC20} from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+
+import "./ICampaign.sol";
+import "./CampaignSecurityModule.sol";
+
+contract Campaign is ICampaign, CampaignSecurityModule {
+    using SafeMath for uint256;
+
+    uint256 public currentCampaign;
+    address public owner;
+
+    mapping(uint256 => ACampaign) public campaigns;
+
+    function initialize(
```

Comment:
Accept child pool address

Reply:
https://github.com/stakeall/cross-chain-staking-shuttle/blob/08f818f2cf07a0aecfc21dec9b46bc1c8a5d831f/contracts/campaign/Campaign.sol#L26

Passed childPool to the initialize